src/game.cpp

```diff
@@ -8095,17 +8085,13 @@ bool npc_menu( npc &who )
         const bool precise = g->u.get_skill_level( skill_firstaid ) * 4 + g->u.per_cur >= 20;
         who.body_window( precise );
     } else if( choice == use_item ) {
-        static const std::string npc_use_flag( "USE_ON_NPC" );
-        const int pos = g->inv_for_filter( _("Use which item:"),[]( const item &it ) {
-            return it.has_flag( npc_use_flag );
-        } );
+        const int pos = g->inv_for_flag( _("Use which item:"), "USE_ON_NPC" );
```

Comment:
Good catch! I swapped them to achieve more consistent definitions, but missed this occurrence. Thank you.